### ISSUE TITLE
Images fade away while using the site with keyboard navigation tools like TAB 

### ISSUE THREAD

**Issue by abhisheknaiidu** [creativecommons/cccatalog-frontend#732]
## Describe the bug
Navigating a site via keyboard means jumping from one focusable element to another in DOM order. While navigating with TAB key through the images, the images fades away, but it comes when we hover over it. 

## To Reproduce
1. Go to https://ccsearch.creativecommons.org/
2. Search for any Item
3. Navigate using TAB key

## Expected behavior

It shouldn't be fade away

## Screenshots

![keyboard](https://user-images.githubusercontent.com/55599878/76184605-951e6b00-61f2-11ea-9fbe-4b1a47ff73ba.png)

## Desktop (please complete the following information)
 - OS: (ex. iOS)
 - Browser : Firefox

## Additional context

Want to work on it, if its a legit issue :)

**Comment by akmadian** [comment]
Thanks for catching this! This should be worked on in our GSoC/ Outreachy project (see #488 for the meta issue). I'm gonna apply the proper labels and close this issue to reduce clutter while still documenting that this task should be looked at.
